packages/blade/src/components/InputGroup/InputRow.web.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import type { InputRowProps } from './types';
+import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
+
+const StyledInputRow = styled.div<{ $gridTemplateColumns: string }>`
+  display: grid;
+  grid-template-columns: ${(props) => props.$gridTemplateColumns};
+`;
+
+export const _InputRow = ({
+  gridTemplateColumns = '1fr',
+  children,
+  testID,
+}: InputRowProps): JSX.Element => {
+  return (
+    <StyledInputRow
+      $gridTemplateColumns={gridTemplateColumns}
+      data-testid={testID}
+      className="input-row"
+    >
+      {children}
+    </StyledInputRow>
+  );
```

Comment:
```suggestion
    <BaseBox
      display="grid"
      gridTemplateColumns={gridTemplateColumns}
      testID={testID}
      className="input-row"
    >
      {children}
    </BaseBox>
  );
```

We can use BaseBox directly for these smaller components right?

Reply:
Done